### PR TITLE
People: Add a button text to the invite button

### PR DESCRIPTION
**Pull request by iamtakashi**
The invite user button doesn't have button text even on desktop size viewport. This PR intends to add a button text of "Invite People" with an icon, "user add". The button text will be hidden with a viewport smaller than 480px.

**Before:**
screen shot 2018-03-07 at 15 52 24

**After:**
screen shot 2018-03-07 at 15 52 48

**<480px:**
screen shot 2018-03-07 at 15 54 01

This PR also changes a button in the invite section from "invite user" to "invite people" to be consistent.

screen shot 2018-03-07 at 15 54 28

### CODE REVIEW

File: client/my-sites/people/people-list-section-header/index.jsx [hunk 1]
```diff
@@ -66,27 +53,10 @@ class PeopleListSectionHeader extends Component {
...
-								context={ this.refs && this.refs.addPeopleButton }
-								position="bottom"
-							>
-								{ translate( 'Invite user', { context: 'button tooltip' } ) }
-							</Tooltip>
-						</Button>
-					</ButtonGroup>
+					<Button compact href={ siteLink } className="people-list-section-header__add-button">
+						<Gridicon icon="user-add" />
+						{ translate( 'Invite People', { context: 'button label' } ) }
```
Comment (a8ci18n) [inline client/my-sites/people/people-list-section-header/index.jsx]: Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'Invite People' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.

**Comment by shaunandrews** [comment]
👍 

I wonder we need to hide the label on mobile, I think there's enough room:

screen shot 2018-03-07 at 12 29 19 pm

**Comment by iamtakashi** [comment]
Thanks, @shaunandrews. I too initially thought not necessary until I saw a "Download Data as CSV" button in the email follower tab...

screen shot 2018-03-07 at 17 35 55

**Comment by drw158** [comment]
I would consider using just "Invite" for the label in the section nav. I think "people" is redundant because you are in the People section, and the icon shows a person.

Not a blocker though.

File: client/my-sites/people/people-list-section-header/index.jsx [hunk 2]
```diff
@@ -17,8 +17,6 @@ import { localize } from 'i18n-calypso';
  */
 import SectionHeader from 'components/section-header';
 import Button from 'components/button';
-import ButtonGroup from 'components/button-group';
-import Tooltip from 'components/tooltip';
```
Comment (drw158) [inline client/my-sites/people/people-list-section-header/index.jsx]: Why the removal of all this code? I wonder why it's there?

Comment (iamtakashi) [reply client/my-sites/people/people-list-section-header/index.jsx]: The old button had a tooltip and was a button group. https://github.com/Automattic/wp-calypso/pull/23085/files#diff-6e27d82feae1a8f5ae795733f6c1ea40L69

**Comment by iamtakashi** [comment]
> I would consider using just "Invite" for the label in the section nav. I think "people" is redundant because you are in the People section, and the icon shows a person.

Thanks, @drw158. That's a good point. I like it shorter, and I think it's clear enough without "people".

**Comment by shaunandrews** [comment]
> I would consider using just "Invite" for the label in the section nav. I think "people" is redundant because you are in the People section, and the icon shows a person.

One of those stupidly obvious things. Great suggestion. I mean, what else are you gonna invite, your cat?

**Comment by iamtakashi** [comment]
OK, let's go with just a word, "Invite".

screen shot 2018-03-07 at 19 42 16

screen shot 2018-03-07 at 19 42 26

File: client/my-sites/people/people-list-section-header/index.jsx [hunk 3]
```diff
@@ -66,27 +53,10 @@ class PeopleListSectionHeader extends Component {
...
-								context={ this.refs && this.refs.addPeopleButton }
-								position="bottom"
-							>
-								{ translate( 'Invite user', { context: 'button tooltip' } ) }
-							</Tooltip>
-						</Button>
-					</ButtonGroup>
+					<Button compact href={ siteLink } className="people-list-section-header__add-button">
+						<Gridicon icon="user-add" />
+						{ translate( 'Invite', { context: 'button label' } ) }
```
Comment (a8ci18n) [inline client/my-sites/people/people-list-section-header/index.jsx]: Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Invite' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.

Comment (michaeldcain) [inline client/my-sites/people/people-list-section-header/index.jsx]: This should probably be a little bit [more descriptive as to intent](https://lamda.blog/2018/03/02/psa-double-check-your-strings-for-ambiguous-contexts/). Maybe `Button to invite more users`.

**Comment by mattsherman** [comment]
Nice improvement!

**Pull request merged**